cli/src/main/java/com/devonfw/tools/ide/tool/gradle/Gradle.java

```diff
@@ -12,6 +18,9 @@
  */
 public class Gradle extends LocalToolCommandlet {
 
+  private static final String BUILD_GRADLE = "build.gradle";
```

Comment:
You also need to support `build.gradle.kts` since `build.gradle` is already the old stuff (legacy) and modern projects switched to kotlin. This should also be supported.

Reply:
I've added a Predicate as suggested.